src/plotApplications/components/map/SingleLeaseMap.js

```diff
@@ -0,0 +1,297 @@
+// @flow
```

Comment:
Um. I read through the files for AreasLayer, PlanUnitsLayer, PlotsLayer and SingleLeaseMap and even made some comments on them, but on closer inspection it seems they're just straight copies of files from another view and aren't used anywhere...